testbed/HIL_Firmware/PDU_HIL/PDU_HIL/include/digitalPot.h

```diff
@@ -0,0 +1,16 @@
+#ifndef DIGITAL_POT_H
+#define DIGITAL_POT_H
+ 
+//check data sheet for AD5260BRUZ200 to retreive values
+#define WIPER_RESISTANCE_OHM 60
+#define MAX_DIGITAL_VALUE 256
+#define NOMINAL_RESISTANCE_OHM 200000
+#define LSB_OHM ((1*NOMINAL_RESISTANCE_OHM)/MAX_DIGITAL_VALUE+WIPER_RESISTANCE_OHM) //841 OHM
+#define POT_MAX NOMINAL_RESISTANCE_OHM-LSB_OHM+WIPER_RESISTANCE_OHM //199,219 OHM
+
+#define PDU_MESSAGE_STATUS 0x8020F03
```

Comment:
I also think that you should keep macros that will only be referenced in the source file at the head of the source file. This means that you are exposing the macro to other files that should never have to use this value